extender/scheduler.go

```diff
@@ -86,22 +87,23 @@ func (m Server) StartServer(port string, certFile string, keyFile string, caFile
 	mx.HandleFunc("/scheduler/filter", handlerWithMiddleware(m.Filter))
 	var err error
 	if unsafe {
-		log.Printf("Extender Listening on HTTP  %v", port)
+		klog.V(2).InfoS("Extender Listening on HTTP "+port, "component", "extender")
 		err = http.ListenAndServe(":"+port, mx)
 	} else {
-		log.Printf("Extender Now Listening on HTTPS  %v", port)
 		srv := configureSecureServer(port, caFile)
 		srv.Handler = mx
-		log.Fatal(srv.ListenAndServeTLS(certFile, keyFile))
+		klog.V(2).InfoS("Extender Listening on HTTPS "+port, "component", "extender")
```

Comment:
Added this line here as no print statement for listening was happening otherwise.